server/src/integration-test/java/com/scalar/db/server/SchemaLoaderIntegrationTestWithScalarDbServer.java

```diff
@@ -6,7 +6,7 @@
 import java.util.Properties;
```

Comment:
Moved the schema loader integration tests with Scalar DB Server to the `server` module.